packages/lit-dev-content/package.json

```diff
@@ -65,23 +66,23 @@
       ],
       "command": "LITDEV_ENV=local NODE_OPTIONS=--experimental-vm-modules eleventy"
     },
-    "build:site:watch": {
+    "build:eleventy:watch": {
       "dependencies": [
-        "build:not-site"
+        "prod:build:assets"
       ],
       "output": [
         "_site"
       ],
-      "#comment": "This is awkward. We want to do the watch part for build:not-site ourselves, but let eleventy do its own watch mode itself. Maybe --incremental will do a quick rebuild of just the changed parts, and we don't need --watch?",
+      "#comment": "This is awkward. We want to do the watch part for build:assets ourselves, but let eleventy do its own watch mode itself. Maybe --incremental will do a quick rebuild of just the changed parts, and we don't need --watch?",
```

Comment:
Should this comment be updated?

Reply:
Not really, same issue still applies.

I think server mode will pretty much solve this, because we can annotate both the actual server, and the built-in `eleventy --watch` command with `"server":true` to indicate that they are long-running, and then have a `dev` script which depends on them both. Then we'll annotate which dependencies of those scripts require a restart (none for `eleventy`, the server's `build:ts` for the server), and then `npm run dev watch` should just do all the right things.

https://github.com/11ty/eleventy/issues/984 is the feature request to support `.tsbuildinfo` style cold start incremental for Eleventy (it's closed, but I think Eleventy does a weird "lodash style" thing where they close all feature requests regardless of whether they will be implemented). This would be even better, since then Wireit could be in control of the timing of when Eleventy re-builds, too.